### PR TITLE
Show a message promoting ad-free to ad-free users

### PR DESCRIPTION
**Pull request by walaura**
## What does this change?
When the proper switch is on, this shows a message for ad-free users informing them they have ad-free. Warm and fuzzy.

## Screenshots
screen shot 2018-10-16 at 4 58 46 pm
(Design & wording is TBD)

## What is the value of this and can you measure success?
Remind our readers of the value of their subscription and of being signed in.

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: common/app/conf/switches/FeatureSwitches.scala [hunk 1]
```diff
@@ -493,4 +493,16 @@ trait FeatureSwitches {
     sellByDate = never,
     exposeClientSide = false
   )
+
+  // Simple & Coherent
+  val ScAdFreeBanner = Switch(
+    SwitchGroup.Feature,
+    "sc-ad-free-banner",
+    "If switched on, ad free users will be told they have ad-free.",
```
Comment (rupertbates) [inline common/app/conf/switches/FeatureSwitches.scala]: nitpick: ad free -> ad-free

**Review by rupertbates — APPROVED** [review]
One tiny nitpick but it looks nice!

File: common/app/conf/switches/FeatureSwitches.scala (hunk 1 above)
Comment (walaura) [reply common/app/conf/switches/FeatureSwitches.scala]: oops!

**Review by GHaberis — APPROVED** [review]

**Comment by LATaylor-guardian** [comment]
👍

**Pull request merged**